src/components/Facets/Facets.tsx

```diff
@@ -91,6 +88,16 @@ export const Facets: FunctionComponent<FacetsProps> = ({
 }: FacetsProps) => {
   const {config} = useStore();
   const translation = useTranslation();
+
+  let i = 0;
+  for (let facet of searchFacets) {
+    if (facet.buckets.length < 1) {
+      searchFacets.splice(i, 1);
+    }
+
+    i++;
```

Comment:
Is this working correctly if there are multiple `facets` with `bucket.length` < 1 ? Because `i` always gets incremented, it might delete the wrong element.

Reply:
It actually was, at least all examples I could find, but just to be sure I replaced this with filter.